### ISSUE TITLE
Authencation Error at GDM Login System Freeze

### ISSUE THREAD

**Issue by Slethen** [pop-os/pop#232]
If this is a bug, please use the template below. If this is a question, or a general discussion topic - please post on our subreddit https://reddit.com/r/pop_os or start a conversation in our chat https://chat.pop-os.org/ - as these are the proper forums for those types of posts.

 **Distribution - (run ```cat /etc/os-release```)**
 NAME="Pop!_OS"
VERSION="18.04 LTS"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Pop!_OS 18.04 LTS"
VERSION_ID="18.04"
HOME_URL="https://system76.com/pop"
SUPPORT_URL="http://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic

 
 **Related Application and/or Package Version - apt policy $PACKAGE NAME**
GDM GNOME Login

**Issue/Bug Description**
Entire OS crashes when clicking the text input for password on GDM.
Unable to access terminal through ctrl+alt+f-key.
The message "Authentication Error" appears.

**Steps to reproduce (if you know)**
Every time I boot and attempt to login

**Expected behavior**
To log in successfully

**Other Notes**
Video: https://photos.app.goo.gl/isSurBgzXGgVju8e8
Fresh install applications installed:
Firefox beta, gimp, sublime, git, protonvpn-cli, inkscape, keepassxc, dropbox, gnome-shell-extension-appindicator, gnome-tweak-tool.

Installed with NVIDIA ISO. AMD FX CPU & NVIDIA GTX 770, 16GB RAM.
Can provide more detail if necessary.

**Comment by cdknight** [comment]
Confirming this error. If you need to get in to the gui, log in to tty2 and startx. Some fonts are off but we have a working environment for the time being.

**Comment by cdknight** [comment]
<code>systemctl set-default multi-user.target</<code> then logging in to the tty with your username and password, and typing <code>startx</code> afterwards is a more permanent solution until this issues gets fixed (just keep sudo apt update && sudo apt upgrade) if you do not want to reinstall a different distro. Once it's fixed, just do <code>systemctl set-default graphical.target</code>. You will notice some of the GNOME shell stylings that pop put in to place are gone. I do not know how to get them back. If you do, let me know.

**Comment by cdknight** [comment]
Not sure but if you just install User Themes extension in <code>gnome-tweak-tool</code> https://extensions.gnome.org/extension/19/user-themes/ you should be able to set the shell theme to pop. Now, I'm able to have a native pop experience and continue using my computer even though this bug is here.

**Comment by Slethen** [comment]
Logging into X with startx and updating glib & pop-session appears to if solved the problem. Although my fonts are broken. Does anyone know Pop's default fonts for:
Window Title
Interface
Document
Monospace?

Thanks @cdknight for the suggestion!

**Comment by cdknight** [comment]
Looks like this bug has been patched. Upon running `sudo apt update && sudo apt upgrade` you should be able to update the `pop-session` package . Then do `systemctl set-default graphical.target` and `reboot` to gdm where you should be able to log in properly.

**Comment by Schyken** [comment]
@cdknight this issue is still occurring for me even after updates, I'm not sure it's been patched yet. Has something changed on your end?

**Comment by cdknight** [comment]
Well, I thought it was patched since both my systems running Pop 18.04 lost the issue after an update to <code>pop-session</code>.

What's your <code> apt-cache policy pop-session</code>?

**Comment by Schyken** [comment]
@cdknight 
pop-session:
  Installed: 2.0.4~1524969709~18.04~8c55250
  Candidate: 2.0.4~1524969709~18.04~8c55250
  Version table:
 *** 2.0.4~1524969709~18.04~8c55250 1001
       1001 http://ppa.launchpad.net/system76/pop/ubuntu bionic/main amd64 Packages
        100 /var/lib/dpkg/status

**Comment by cdknight** [comment]
And the initial startx from tty2 works for you? Or do you have to log in as root to start x?

**Comment by cdknight** [comment]
https://www.reddit.com/r/pop_os/comments/8fmvtk/authentication_error_on_desktop_when_upgrading_to/ This error is on reddit too.

**Comment by Schyken** [comment]
It works without having to log in as root to start x, yeah.

**Comment by cdknight** [comment]
What about logging in with the Not listed? I think that worked on my laptop but not on my desktop.

On Apr 29, 2018 4:58 PM, Schyken <notifications@github.com> wrote:

It works without having to log in as root to start x, yeah.

**Comment by Schyken** [comment]
@cdknight Nah, that doesn't do anything for me. It's interesting because it __only__ happens after I've already logged in as a user and then log in as another. It never happens initially.

**Comment by cdknight** [comment]
Oh, I see. I haven't tested that yet because in most distros that feature is buggy for me and I don't use it anyway.

**Comment by Schyken** [comment]
Makes sense, though this isn't specific to switching users, the primary user can be logged out in this case, it's just as long as a user was logged in being followed by another user logging in, whether the initial user logged out or not seems to be irrelevant on my system. 

Not really sure what causes this, but it's definitely an interesting quirk. :3

**Comment by Schyken** [comment]
Any word on this issue? I've noticed this in Ubuntu as well, which suggests it's not isolated to Pop!_OS. Does anyone know what causes this?

**Comment by cdknight** [comment]
I'm not sure.  I tested on a multi user system upon logging out you cannot log in to a different user without restarting. I can test other distros, though, and will report back soon. Maybe there is a different configuration for GDM on Ubuntu-based distros. Would be interesting to see if it happens on Debian, but maybe there is a way to fix it by configuring GDM.

**Comment by mmstick** [comment]
@Schyken Can you confirm if this is fixed in 18.10?

@cdknight Did you ever try this out in other distros?

I suspect either a Debian or Ubuntu patch is causing the issue. Anyone reported this to Launchpad?
